include/object/file.hpp

```diff
@@ -39,8 +39,7 @@ namespace ntw::obj {
                 NTW_INLINE ~disposer() { static_cast<void>(unique_file::destroy(_path)); }
             };
 
-            NTW_INLINE basic_file()  = default;
-            NTW_INLINE ~basic_file() = default;
```

Comment:
I am OK with this getting deleted however this line has a purpose of forcing the delete to get inlined which didn't happen without the NTW_INLINE